src/components/widgets/accounts-invoices-list/accounts-invoices-list.directive.coffee

```diff
@@ -0,0 +1,66 @@
+module = angular.module('impac.components.widgets.accounts-invoices-list', [])
+module.controller('WidgetAccountsInvoicesListCtrl', ($scope, $q, ImpacRoutes, BoltResources) ->
+
+  # == Context and Helpers ========================================================================
+  w = $scope.widget
+  bolts = ImpacRoutes.bolts()
+  bolt_path = _.find(bolts, {name: 'finance', provider: 'maestrano'}).path
+
+  $scope.trxList = {
+    display: false,
+    resources: 'invoices',
+    transactions: [],
+    params: {}
+  }
+
+  # Widget Settings --------------------------------------
+  $scope.orgDeferred = $q.defer();
+
+  settingsPromises = [$scope.orgDeferred.promise]
+
+  # Widget specific methods
+
+  $scope.trxList.show = ->
+    $scope.trxList.display = true
+
+  $scope.trxList.hide = ->
+    $scope.trxList.display = false
+
+  $scope.trxList.fetch = (currentPage = 1) ->
```

Comment:
Quick question: do you think this could be moved to the transactions-list component itself?

Reply:
I don't think so, because sometimes the show or hide method may require extra logic. For instance in the accounts-cash-projection widget: 
```
  $scope.trxList.show = ->
    $scope.trxList.display = true

  $scope.trxList.hide = ->
    $scope.trxList.display = false
    if $scope.trxList.updated
      ImpacWidgetsSvc.show(w).then(-> $scope.trxList.updated = false)
```